model/model_training/models/patching.py

```diff
@@ -8,15 +8,15 @@
 import torch.nn as nn
 import torch.nn.functional as F
 from torch.nn.utils.rnn import pad_sequence
-from transformers import GPTNeoXForCausalLM, GPTNeoXModel, LlamaForCausalLM, LlamaModel
+from transformers import GPTNeoXForCausalLM, GPTNeoXModel#, LlamaForCausalLM, LlamaModel
```

Comment:
is there any trouble with uncommenting the lama models here and in the changes below? should work as far as I can see